lib/tokenize.cpp

```diff
@@ -9447,7 +9447,7 @@ void Tokenizer::simplifyBitfields()
             !Token::Match(tok->next(), "case|public|protected|private|class|struct") &&
             !Token::simpleMatch(tok->tokAt(2), "default :")) {
             Token *tok1 = (tok->next()->str() == "const") ? tok->tokAt(3) : tok->tokAt(2);
-            if (Token::Match(tok1, "%name% : %num% ;"))
+            if (tok1 && Token::Match(tok1, "%name% : %num% ;"))
```

Comment:
This change will have no effect. `Token::Match()` returns `false` if it's passed a null `Token*` pointer.